Client/Client.cs

```diff
@@ -114,67 +117,92 @@ public DeltaQueryResult DeltaQuery(string stateToken, string entitySet)
         /// </param>
         /// <param name="propertyList">List of properties to retrieve.</param>
         /// <returns>Result from the Delta Query service.</returns>
-        public DeltaQueryResult DeltaQuery(
-            string entitySet,
+        public async Task<DeltaQueryResult> DeltaQueryAsync(
```

Comment:
We can talk about this on Monday.  The original sample is actually trying to be more than just a sample and it is trying to create a generic client that can query any arbitrary entityset for deltaquery results.  That type of functionality belongs in our core SDK, not in a sample.  So, I agree with what you are doing here by using graphServiceClient to demo doing a DeltaQuery on a single entityset.  However, to reduce the confusion, I think we should lift this code out of the Client class into the program class.

Reply:
agreed, this sample is a little too confusing.